python/GangaDirac/test/Unit/Files/TestDiracFile.py

```diff
@@ -130,21 +130,21 @@ def test_get(df):
 
     df.lfn = 'lfn'
     with patch('GangaDirac.Lib.Files.DiracFile.execute', return_value={'OK': True, 'Value': {'Successful': {'%s' % df.lfn: True}}}) as execute:
-        assert df.get() is None
+        assert df.get() is True
```

Comment:
I'll add some lines to the test to describe the change in return then.
My concern with this is the class could change to return a non bool which is _not_ the direction we want to go in development wise but will still pass an ambiguous assert. I'd much rather be explicit here about what we expect.